core/templates/dev/head/css/oppia.css

```diff
@@ -2767,6 +2767,19 @@ div#ng-curtain {
   display: inline-block;
 }
 
+.oppia-notification-clickable {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  width: 100%;
+  height: 100%;
```

Comment:
I'm guessing 
```css
bottom: 0;
left: 0;
right: 0;
top: 0;
```
should also do the trick (instead of using `width: 100%` and `height: 100%`) ?